modules/users/manifests/anacastillobotto.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the anacastillobotto user
+class users::<anacastillobotto> {
+  govuk_user { '<anacastillobotto>':
```

Comment:
Hi Ana, your class and username needs to be without the angled brackets. Please refer to https://github.com/alphagov/govuk-puppet/pull/12069/commits/1b9a267dc60791f8594ff00d38331a66739f3b84 if it helps!